internal/consensus/reactor.go

```diff
@@ -359,8 +359,12 @@ func (conR *Reactor) Receive(e p2p.Envelope) {
 			ps.SetHasVoteFromPeer(msg.Vote, height, valSize, lastCommitSize)
 
 			cs.peerMsgQueue <- msgInfo{msg, e.Src.ID(), time.Time{}}
-		// case *CommitMessage:
-		// TODO Send it to the queue like VoteMessage
+		case *CommitMessage:
+			cs := conR.conS
+			cs.Logger.Info("XX Received ")
+			// TODO setHasAggregatedVoteFromPeer
```

Comment:
???

Reply:
I mainly wanted to indicated that eventually we should mark that we have  received a commit from a peer and not request it again. But as this is catch-up maybe it is not needed?